devops/resources/run-ansible.sh

```diff
@@ -7,7 +7,7 @@ pip install -r requirements.txt
 pip install awscli
 
 env
-
+export EC2_CACHE_PATH="ec2-cache"
```

Comment:
does this directory exist?  Does it need to?  '.' is a special unix directory meaning the current directory, so it's guaranteed to exist.  You can also use mktemp to generate a temp directory for each workspace, but '.' had a nice side-effect of being managed by the jenkins workspace for each assumed role/job.